app/views/components/_filter_summary.html.erb

```diff
@@ -0,0 +1,40 @@
+<% add_app_component_stylesheet("filter-summary") %>
+
+<%
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  filters ||= []
+
+  clear_all_href ||= nil
+  clear_all_text ||= nil
+  heading_level ||= 3
+  heading_text ||= nil
+
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+  component_helper.add_class(shared_helper.get_margin_bottom) if local_assigns[:margin_bottom]
+  component_helper.add_class("app-c-filter-summary")
+%>
+
+<% if filters && filters.length > 0  %>
+  <%= tag.div(**component_helper.all_attributes) do %>
+    <% if heading_text.present? %>
+    <%= content_tag("h#{heading_level}", class: "app-c-filter-summary__heading") do %>
+      <% heading_text %>
+    <% end %>
+    <% end %>
+    <% filters.each do |filter| %>
```

Comment:
IMHO this is a list and we probably want to semantically have it as such (in an `ul`).